src/components/modals/ReminderModal.vue

```diff
@@ -82,6 +82,31 @@ export default {
           }))
         ];
       });
+
+      // add out of script traveller reminders
+      this.$store.state.extraTravellers.forEach(role => {
```

Comment:
`travelers` is spelt with one L throughout the whole app, so I'd like to keep it consistent here as well. 😬